pkgs/development/python-modules/mdformat-wikilink/default.nix

```
@@ -0,0 +1,43 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  mdformat,
+  poetry-core,
+  pytest-cov,
+  pytestCheckHook,
+  pythonOlder,
+}:
+
+buildPythonPackage rec {
+  pname = "mdformat-wikilink";
+  version = "0.2.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "tmr232";
+    repo = "mdformat-wikilink";
+    rev = "v${version}";
+    hash = "sha256-KOPh9iZfb3GCvslQeYBgqNaOyqtWi2llkaiWE7nmcJo=";
+  };
+
+  nativeBuildInputs = [
+    poetry-core
+    pytest-cov
+  ];
+
+  propagatedBuildInputs = [ mdformat ];
```

```suggestion
  dependencies = [
    markdown-it-py
    mdformat
  ];
```
https://github.com/tmr232/mdformat-wikilink/blob/257c712c705fc263cb6afaf1238a4e16f098628b/pyproject.toml#L10-L13